### PR TITLE
Custom timeout

### PR DESCRIPTION
**Pull request by Loghorn**
This allows to pass a custom timeout to each toast, overriding the default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Create customized toast component.
 - `id` Auto generated message ID
 - `text` Text of the toast message
 - `type` Type of the toast message
+- `dismissAfter` Optional: ovverride default dismissInterval
```
Comment (ktsn) [inline README.md]: override?

**Comment by ktsn** [comment]
Hi, @Loghorn. Thanks for your PR! This looks nice feature.
But there are some unnecessary diffs like changing indent and code styling. Can we reverse them?

File: README.md (hunk 1 above)
Comment (Loghorn) [inline README.md]: You are tottally correct ;)

**Comment by Loghorn** [comment]
Of course we can. Sorry about that, my editor configuration automatically reformat code on save.

**Comment by Loghorn** [comment]
Is it better now? 😄

**Comment by ktsn** [comment]
Can we also revert version number? because I'll update that by npm command ;)

**Comment by Loghorn** [comment]
Done

**Comment by ktsn** [comment]
Thanks!

**Pull request merged**